tokens/elevation-shadows/box-shadow/box-shadow.tokens.json

```diff
@@ -0,0 +1,114 @@
+{
+  "box-shadow": {
+    "down": {
+      "1": {
+        "value": "0 .0625rem .125rem rgba(0, 0, 0, .15), 0 .0625rem .25rem rgba(0, 0, 0, .15)",
```

Comment:
Probably we need `"themable": true` here too

Reply:
done